packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -1,24 +1,129 @@
 @import '../../common/common';
 
-.root {
-  user-select: none;
-  margin-bottom: 8px;
-}
+.row, .column {
+  .root {
+    user-select: none;
+  }
+
+  .titleLabel {
+      padding: 0;
+  }
 
-.titleLabel {
-  padding: 0;
+  .line {
+    display: flex;
+    position: relative;
+  }
+
+  .line span {
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+
+  .slideBox {
+    @include focus-border();
+    background: transparent;
+    border: none;
+    padding: 0;
+    margin: 0;
+    flex-grow: 1;
+    line-height: 28px;
+  }
+
+  .slideBox .thumb {
+    border: 2px solid $ms-color-neutralSecondary;
+    box-sizing: border-box;
+    background: $ms-color-white;
+    display: block;
+    width: 16px;
+    height: 16px;
+    position: absolute;
+    border-radius: 10px;
+  }
+
+  .container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .valueLabel {
+    flex-shrink: 1;
+    width: 30px;
+    line-height: 1;
+  }
 }
 
-.line {
-  display: flex;
-  position: relative;
-  width: 100%;
+.row {
+  .root {
+    margin-bottom: 8px;
+  }
+
+  .line {
+    width: 100%;
+  }
+
+  .line span {
+    height: 4px;
+  }
+
+  .slideBox {
+    height: 28px;
+    padding: 0 8px; // Make room for thumb at ends of line
+  }
+
+  .slideBox .thumb {
+    top: -6px;
+    @include transform-translateX(-50%);
+  }
+
+  .valueLabel {
+    margin: 0 8px;
+  }
 }
 
-.line span {
-  height: 4px;
-  border-radius: 4px;
-  box-sizing: border-box;
+.column {
+  .root {
+    margin-left: 8px;
+    margin-right: 8px;
+  }
+
+  .line {
+    height: 100%;
+    width: 4px;
+    margin: 0 auto;
+    flex-direction: column-reverse;
+  }
+
+  .line span {
```

Comment:
avoid using elements in selectors. Put a classname on the thing you're trying to target.

Reply:
Hi, David. I have do some changes. Could you help to review it ? Thank you so much!